src/Compiler/Checking/import.fsi

```diff
@@ -51,6 +51,26 @@ type ImportMap =
     /// The TcGlobals for the import context
     member g: TcGlobals
 
+module Nullness =
```

Comment:
We don't often put the types in a nested module, though it's no big deal, as yo ulike